source/support.tex

```diff
@@ -2877,19 +2877,19 @@
 in storage occupied by an existing object of the same type,
 a pointer to the original object
 can be used to refer to the new object
-unless the type contains \tcode{const} or reference members;
-in the latter cases,
+unless its complete object is a const object;
+in the latter case,
 this function can be used to obtain a usable pointer to the new object.
 See~\ref{basic.life}.
 \end{note}
 
 \pnum
 \begin{example}
 \begin{codeblock}
-struct X { const int n; };
-X *p = new X{3};
+struct X { int n; };
+const X *p = new const X{3};
 const int a = p->n;
-new (p) X{5};                       // \tcode{p} does not point to new object\iref{basic.life} because \tcode{X::n} is \tcode{const}
+new (const_cast<X*>(p)) const X{5};  // \tcode{p} does not point to new object\iref{basic.life} because its type is \tcode{const}
```

Comment:
```suggestion
new (const_cast<X*>(p)) const X{5};    // \tcode{p} does not point to new object\iref{basic.life} because its type is \tcode{const}
```

Reply:
Fixed.